jx-requirements.yml.tpl

```
@@ -11,19 +11,15 @@ spec:
       secretStorage:
         keyVaultName: ${key_vault_name}
   %{ endif }
-  %{ if dns_enabled }
       dns:
         resourceGroup: ${dns_resource_group}
         tenantId: ${dns_tenant_id}
         subscriptionId: ${dns_subscription_id}
-  %{ endif }
```

```suggestion
  %{ if domain_enabled }
      dns:
        resourceGroup: ${dns_resource_group}
        tenantId: ${dns_tenant_id}
        subscriptionId: ${dns_subscription_id}
  %{ endif }
```

Not sure but this part seems to be only required when the domain is enabled.

---

Looks like this isn't done within the gcp one: https://github.com/jenkins-x/terraform-google-jx/blob/master/modules/jx-requirements-v3.yml.tpl#L10

Can add it but it's up to you

---

Indeed, there isn't any "dns" entry in GCP jx-requirements.

But as there is one for Azure, I think we should keep it declared only when the domain is enabled. I don't know what would happen if it's declared without any domain enabled.